torchdata/datapipes/iter/util/rararchiveloader.py

```diff
@@ -107,7 +107,9 @@ def __iter__(self) -> Iterator[Tuple[str, io.BufferedIOBase]]:
                 inner_path = os.path.join(path, info.filename)
                 file_obj = rar.open(info)
 
-                yield inner_path, StreamWrapper(file_obj)  # type: ignore[misc]
+                yield inner_path, StreamWrapper(file_obj, stream)  # type: ignore[misc]
```

Comment:
nit: Should we pass in the `name` here to `StreamWrapper`? Or there is a good reason to leave it out?

Reply:
There is no reason to skip names, they are convenient for debugging. Will add in both cases.